packages/client/screens/NotificationsScreen.tsx

```diff
@@ -0,0 +1,56 @@
+import { View } from "../components/Themed";
+import React from "react";
+import { StyleSheet, Switch } from "react-native";
+import { AuthContext } from "../components/contexts/AuthContext";
+import { UsersGateway } from "../gateways/users_gateway";
+import { HStack, VStack, Text } from "native-base";
+import { farArrowRightArrowLeft } from "../components/icons/regular/farArrowRightArrowLeft";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+
+export default function NotificationsScreen() {
+    const { token, user } = React.useContext(AuthContext);
+    const usersGateway = new UsersGateway();
+
+    const [isEnabled, setIsEnabled] = React.useState(user.areNotificationsEnabled);
+    async function toggleSwitch() {
+        usersGateway.update(
+            {
+                userId: user.id,
+                areNotificationsEnabled: !isEnabled,
+            },
+            token,
+        );
+
+        user.areNotificationsEnabled = !isEnabled;
+        setIsEnabled(!isEnabled);
+    }
+
+    return (
+        <View style={styles.view}>
+            <HStack alignItems={"center"} space={19}>
```

Comment:
For some reason native base styling doesn't treat space amount as pixels so we have to set it manually.

```suggestion
            <HStack alignItems={"center"} space={"15px"}>
```